storage/api.go

```diff
@@ -46,6 +47,16 @@ func (b *QueryBatch) Queue(cmd string, args ...interface{}) {
 	})
 }
 
+// Append merges another batch into the current batch.
+func (b *QueryBatch) Append(qb *QueryBatch) {
```

Comment:
Nit: I would prefer to follow py nomenclature here, or at least not clash with it. In py, `.append(el)` is for appending a single element, and `.extend(lst)` is for appending all the elements of `lst`. Similarly in go, `append()` expects each element-to-be-appended as a a separate argument. So maybe `Extend` or `ExtendWith` here?